scripts/generate-certs.sh

```diff
@@ -9,7 +9,7 @@ case "$(uname -s)" in
      ;;
    Linux)
     mkdir -p ../certs
-    openssl req -nodes -new -x509 -keyout ../certs/key.pem -out ../certs/cert.pem
+    openssl req -nodes -new -x509 -keyout ../certs/key.pem -out ../certs/cert.pem ./opensslcnf
```

Comment:
Which domain these certificates are generated for? What about certificate for minio?

https://etherealengine.github.io/etherealengine-docs/docs/host/devops_deployment/microk8s_windows#accept-invalid-certs